app/helpers/local_time_helper.rb

```diff
@@ -67,4 +53,33 @@ def extract_options_and_value(options, value_key = nil)
         [ {}, options ]
       end
     end
+
+    def find_time_format(format)
+      if format.is_a?(Symbol)
+        find_time_format_by_name(format)
+      else
+        [format.presence || LocalTime.default_time_format, nil]
```

Comment:
Ah, thanks! I'll fix this formatting here and add rubocop in a separate PR to avoid polluting the diffs here. I checked and rubocop would touch more than a couple files — all with minor, changes.